src/contractTest/java/uk/gov/hmcts/reform/managecase/consumer/CCDConsumerTest.java

```diff
@@ -0,0 +1,201 @@
+        HttpHeaders.AUTHORIZATION, IDAM_OAUTH2_TOKEN,
+        SERVICE_AUTHORIZATION, SERVICE_AUTHORIZATION_TOKEN
+    );
+
+    private static final String CASE_ASSIGNMENT_REQUEST = "{"
+        + "\"case_users\": ["
+                + " {\"organisation_id\": \"Test_Org\", \"case_id\": \"1588234985453946\", \"user_id\": \"abc123\","
+                + " \"case_role\": \"[Collaborator]\" }"
+            + "]"
+        + "}";
+
+    @Pact(provider = "ccd", consumer = "mca_consumer")
+    public RequestResponsePact getCaseAssignments(PactDslWithProvider builder) throws Exception {
+        return builder
+            .given("CCD successfully return case assignments")
+            .uponReceiving("Request to get case assignments")
+                .path("/case-users")
+                .method(HttpMethod.GET.toString())
+                .matchQuery("case_ids", "^\\d{16,16}$", "1588234985453946")
+                .matchQuery("user_ids", "[a-zA-Z0-9-]+", "33dff5a7-3b6f-45f1-b5e7-5f9be1ede355")
+                .headers(headers)
+            .willRespondWith()
+                .status(HttpStatus.OK.value())
+                .body(
+                    new PactDslJsonBody()
+                        .minArrayLike("case_users", 1, 1)
+                        .stringType("case_id", "1588234985453946")
+                        .stringType("user_id", "33dff5a7-3b6f-45f1-b5e7-5f9be1ede355")
+                        .stringType("case_role", "[Collaborator]")
+                        .closeArray()
+                )
+            .toPact();
+    }
+
+    @Pact(provider = "ccd", consumer = "mca_consumer")
+    public RequestResponsePact getCaseAssignmentsWhenCaseIdsPassed(PactDslWithProvider builder) throws Exception {
+        return builder
+            .given("CCD successfully return case assignments")
+            .uponReceiving("Request to get case assignments")
+            .path("/case-users")
+            .method(HttpMethod.GET.toString())
+            .headers(headers)
+            .willRespondWith()
+            .status(HttpStatus.BAD_REQUEST.value())
+            .body(
+                new PactDslJsonBody()
+                    .stringValue("message", "Case ID list is empty")
+            )
+            .toPact();
+    }
+
+    @Pact(provider = "ccd", consumer = "mca_consumer")
+    public RequestResponsePact addCaseAssignments(PactDslWithProvider builder) throws Exception {
+        return builder
+            .given("CCD successfully add case assignments")
+            .uponReceiving("Request to add case assignments")
+            .path("/case-users")
+            .method(HttpMethod.POST.toString())
+            .body(CASE_ASSIGNMENT_REQUEST, ContentType.APPLICATION_JSON)
+            .headers(headers)
+            .willRespondWith()
+            .status(HttpStatus.CREATED.value())
+            .body(
+                new PactDslJsonBody()
+                    .stringValue("message", "Case-User-Role assignments created successfully")
```

Comment:
is this the message returned from data store?
"Case-User-Role assignments created successfully"